service/src/test/java/bio/terra/catalog/datarepo/DatarepoClientTest.java

```diff
@@ -0,0 +1,43 @@
+package bio.terra.catalog.datarepo;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import bio.terra.catalog.config.DatarepoConfiguration;
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.datarepo.client.auth.OAuth;
+import org.junit.jupiter.api.Test;
+
+class DatarepoClientTest {
+
+  private static final String BASE_PATH = "basepath";
+  private static final String TOKEN = "token";
+
+  private final DatarepoClient client;
+
+  DatarepoClientTest() {
+    client = new DatarepoClient(new DatarepoConfiguration(BASE_PATH));
+  }
+
+  @Test
+  void testApis() {
+    var user =
+        new AuthenticatedUserRequest.Builder()
+            .setEmail("")
+            .setSubjectId("")
+            .setToken(TOKEN)
+            .build();
+    var snapshotsClient = client.snapshotsApi(user).getApiClient();
```

Comment:
should we just expose getApiClient() so we don't have to create a snapshotsApi to then get its ApiClient?

Reply:
This particular form is only used in test code, as a (very artificial) way to test the created APIs. Since it's not used in non-test code, I don't think we should add an API in `client` for this.